Tasks/SonarQubePostTest/SonarQubeBuildBreaker.ps1

```diff
@@ -0,0 +1,176 @@
+. ./SonarQubeHelper.ps1
+
+$DefaultNumRetriesForAnalysisToComplete = 120; # there 1s delay between each retry plus the time it takes to make a web call
```

Comment:
Should have made this clearer: this is configurable via a build variable - so we don't want to expose this to the user, but in case someone hits this timeout we have a way of unblocking them. 120 is the default if the build variable is not set.